### PR TITLE
Use new etcd AMI

### PR DESCRIPTION
**Pull request by gargravarr**
This **Pull Request (PR)** updates the etcd CloudFormation template to support the new etcd AMI. The new AMI is also set as the default for etcd.

Merging this PR _won't rotate_ the etcd instances. The effective rotation will still be done manually using [AWS Instance Refresh](https://docs.aws.amazon.com/autoscaling/ec2/userguide/asg-instance-refresh.html).

### CODE REVIEW

**Comment by gargravarr** [comment]
:+1:

File: cluster/etcd/stack.yaml [hunk 1]
```diff
@@ -78,7 +75,7 @@ Resources:
         Timeout: PT15M
     Properties:
       DesiredCapacity: {{.Cluster.ConfigItems.etcd_instance_count}}
-      HealthCheckGracePeriod: 300
+      HealthCheckGracePeriod: 0
```
Comment (demonCoder95) [inline cluster/etcd/stack.yaml:78]: Any specific reason to set this to 0? Are we relying on the `HeartbeatTimeout` in `AutoscalingLifecycleHook` instead? 🤔

Comment (gargravarr) [reply cluster/etcd/stack.yaml:78]: Yes, exactly that :slightly_smiling_face:

**Comment by demonCoder95** [comment]
👍

**Pull request merged**